test/test.js

```diff
@@ -20,11 +20,11 @@ describe("loader", function() {
 		}, "").should.be.eql(HEADER +
 			// First import
 			"var abc = (abc || {});\n" +
-			"abc.def = {};\n" +
+			"abc.def = abc.def || {};\n" +
```

Comment:
Maybe put this in a additional test, instead of changing the existing one, if this makes no difference and covers the purpose of the current test, feel free to ignore me here :)